rocketpy/plots/dispersion_plots.py

```diff
@@ -0,0 +1,177 @@
+__author__ = (
+    "Mateus Stano Junqueira, Guilherme Fernandes Alves, Bruno Abdulklech Sorban"
+)
+__copyright__ = "Copyright 20XX, RocketPy Team"
+__license__ = "MIT"
+
+
+import matplotlib.pyplot as plt
+
+from ..tools import generate_dispersion_ellipses
+
+
+class _DispersionPlots:
+    """Class to plot the dispersion results of the dispersion analysis."""
+
+    def __init__(self, dispersion):
+        self.dispersion = dispersion
+        return None
+
+    def plot_ellipses(
```

Comment:
We don't need to have the word 'plot' in the methods of _DispersionPlots, or it ends up redundant with `Dispersion.plot.plot_ellipses`. This also applies to the prints

Reply:
solved with commit f43b825bda4443de608de936a4968b62b5f7b35b